doc/source/whatsnew/v1.1.0.rst

```diff
@@ -37,6 +37,37 @@ For example:
    ser.loc["May 2015"]
 
 
+.. _whatsnew_110.dataframe_or_series_comparing:
+
+Comparing two `DataFrame` or two `Series` and summarizing the differences
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We've added :meth:`DataFrame.compare` and :meth:`Series.compare` for comparing two `DataFrame` or two `Series` (:issue:`30429`)
```

Comment:
can you add a similar example in in reshaping.rst or merge.rst (with a link from the other section) and a link here

Reply:
Added, can you review?

I have chosen to add the examples in merge.rst, as this `compare` method is more similar to a merging operation in that they both involved two aligned objects, and somehow combine them both.